index.html

```diff
@@ -16,17 +16,109 @@
+            <h1>Multi-Post Stories</h1>
+            <div><img src="./images/Frame 2.png" alt=""></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tags">
+                <li><img src="./images/htmltag.png" alt=""></li>
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+        <div class="cards" id="cardthird">
+            <div class="shiny"><img src="./images/shinyorange.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Tonic</h1>
+            <div><img src="./images/Frame 2.png" alt=""></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tags">
+                <li><img src="./images/htmltag.png" alt=""></li>
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+        <div class="cards" id="cardfourth">
+            <div class="white"><img src="./images/white.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Multi-Post Stories</h1>
+            <div><img src="./images/Frame 2.png" alt=""></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tags">
+                <li><img src="./images/htmltag.png" alt=""></li>
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+      <h1>About Myself</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <a href="#">LET'S CONNECT</a>
+      <ul class="icons2">
+          <li><a href="https://www.freecodecamp.org/"> <img src="./images/Githubicon.png" alt=""> </a></li> 
+          <li><a href="https://www.linkedin.com/in/adri%C3%A1n-ju%C3%A1rez-17a51b263/"> <img src="./images/Linkedinicon.png" alt=""> </a></li>
+              <li><a href="https://www.theodinproject.com/"> <img src="./images/Unknownicon.png" alt=""> </a></li> 
+        <li><a href="https://twitter.com/CryptogodAdrian"> <img src="./images/Twittericon.png" alt=""> </a></li> 
+        <li><a href="https://www.w3schools.com/"> <img src="./images/Mediumicon.png" alt=""> </a></li> 
+     </ul>
+     <div class="get">Get my resume</div>
```

Comment:
- [ ] The button (e.g., "Get my resume") is a link `a` or `button` tag,
- Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href.